infra/base-images/base-runner/run_fuzzer

```diff
@@ -25,7 +25,12 @@ FUZZER=$1
 shift
 
 CORPUS_DIR="/tmp/${FUZZER}_corpus"
-FUZZER_OUT="/tmp/${FUZZER}_out"
+
+if [[ "$RUN_FUZZER_MODE" = interactive ]]; then
+  FUZZER_OUT="$OUT/${FUZZER}_${FUZZING_ENGINE}_${SANITIZER}_out"
+else
+  FUZZER_OUT="/tmp/${FUZZER}_out"
```

Comment:
for consistency could you please make this the same format (${FUZZER}_${FUZZING_ENGINE}_${SANITIZER}_out) ?

Reply:
Thank you for the review! I hesitated to change the path because I thought that the path somehow used somewhere else, but if it's safe, I'll definitely change it.

Reply:
@oliverchang I'm not sure whether github sends notifications when a conversation is marked as "resolved" so this comment is just to let you know that I changed the code as you suggested.